docs/install.md

```diff
@@ -123,34 +122,10 @@ under the [Apache 2.0 License][apache2l].
 [examples]: https://github.com/tektoncd/pipeline/tree/main/examples/
 [cca4]: https://creativecommons.org/licenses/by/4.0/
 [apache2l]: https://www.apache.org/licenses/LICENSE-2.0
-[post-install]: ./additional-configs.md
-
-## Troubleshooting
```

Comment:
why remove this? the commit message says it's moved but I just see it deleted

Reply:
It's still there, it points to the absolute website location now (https://tekton.dev/docs/pipelines/additional-configs/). The sync script wasn't parsing it correctly for some reason.

Reply:
On second thoughts, I converted the PR to draft. I'm going to make sure all the links work for the versioned docs.

Reply:
I just realized. Did you mean the troubleshooting section? Sorry, I inadvertently went completely off on a tangent. I updated the description. There's a comment about that in the [PR that added it](https://github.com/tektoncd/pipeline/pull/6121#issuecomment-1433078771). 

Covering the the setup process for Kind (or minikube for that matter) is out of the scope of the Tekton docs, and instead we link to the the tools' official docs and to some tekton-specific configs.  We had [a similar conversation](https://github.com/tektoncd/website/pull/451#pullrequestreview-1133151770) when I first wrote the [local installation docs](https://tekton.dev/docs/installation/local-installation/).

Reply:
Thanks! can you just update the commit/PR title to reflect that this also removes troubleshooting docs?

Reply:
Updated